stock_barcodes_advanced/views/stock_picking_views.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="stock_picking_kanban" model="ir.ui.view">
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock_barcodes.stock_picking_kanban" />
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='action_barcode_scan']" position="attributes">
+                <attribute name="states">confirmed</attribute>
```

Comment:
Please add as a comment replaced attribute content. And set a lower priority for this view (as a "replace" is made)